sdk/appconfiguration/azure-data-appconfiguration/README.md

```diff
@@ -167,61 +167,115 @@ ConfigurationAsyncClient client = new ConfigurationClientBuilder()
-        .buildClient();
-client.setSetting("some_key", "some_label", "some_value");
-ConfigurationSetting setting = client.getSetting("some_key", "some_label");
+ConfigurationSetting setting = client.setConfigurationSetting("some_key", "some_label", "some_value");
+ConfigurationSetting retrievedSetting = client.getConfigurationSetting("some_key", "some_label");
+```
+For coditional request,
+```Java
+Response<ConfigurationSetting> settingResponse = client.getConfigurationSettingWithResponse(setting, null, true, Context.NONE);
 ```
 
 ### Update an existing Configuration Setting
 
-Update an existing Configuration Setting by calling setSetting.
+Update an existing Configuration Setting by calling setConfigurationSetting.
 ```Java
-ConfigurationClient client = new ConfigurationClientBuilder()
-        .connectionString(connectionString)
-        .buildClient();
-client.setSetting("some_key", "some_label", "some_value");
-ConfigurationSetting setting = client.setSetting("some_key", "some_label", "new_value");
+ConfigurationSetting setting = client.setConfigurationSetting("some_key", "some_label", "some_value");
+ConfigurationSetting updatedSetting = client.setConfigurationSetting("some_key", "some_label", "new_value");
+```
+For coditional request,
+```Java
+Response<ConfigurationSetting> settingResponse = client.setConfigurationSettingWithResponse(setting, true, Context.NONE);
 ```
 
 ### Delete a Configuration Setting
 
-Delete an existing Configuration Setting by calling deleteSetting.
+Delete an existing Configuration Setting by calling deleteConfigurationSetting.
 ```Java
-ConfigurationClient client = new ConfigurationClientBuilder()
-        .connectionString(connectionString)
-        .buildClient();
-client.setSetting("some_key", "some_label", "some_value");
-ConfigurationSetting setting = client.deleteSetting("some_key", "some_label");
+ConfigurationSetting setting = client.setConfigurationSetting("some_key", "some_label", "some_value");
+ConfigurationSetting deletedSetting = client.deleteConfigurationSetting("some_key", "some_label");
+```
+For coditional request,
+```Java
+Response<ConfigurationSetting> settingResponse = client.deleteConfigurationSettingWithResponse(setting, true, Context.NONE);
+```
+
+### List Configuration Settings with Multiple Keys
+
+List multiple configuration settings by calling listConfigurationSettings.
+Pass null SettingSelector into the method If you want to fetch all configuration settings.
+```Java
+String key = "some_key";
+String key2 = "new_key";
+client.setConfigurationSetting(key, "some_label", "some_value");
+client.setConfigurationSetting(key2, "new_label", "new_value");
+SettingSelector selector = new SettingSelector().setKeys(key, key2);
+PagedIterable<ConfigurationSetting> settings = client.listConfigurationSettings(selector);
+```
+
+### List Revisions of Multiple Configuration Settings
+
+List all revision of a configuration settings by calling listRevisions.
```

Comment:
When you invoke or are referring to code, I'd suggest using a backtick, so it's apparent to readers. (ie. `listRevisions`)